credsweeper/rules/config.yaml

```diff
@@ -1,1013 +1,1219 @@
-  min_line_len: 29
-  required_substrings:
-    - GR1348941
-
-- name: Gitlab Registration Runner Token 2023
-  severity: high
-  confidence: strong
-  type: pattern
-  values:
-    - (?<![0-9A-Za-z_+-])(?P<value>glrt-[a-zA-Z0-9_-]{20})(?![=0-9A-Za-z_+-])
-  filter_type: TokenPattern
-  min_line_len: 25
-  required_substrings:
-    - glrt-
-
-- name: Grafana Provisioned API Key
-  severity: high
-  confidence: strong
-  type: pattern
-  values:
-    - (?<![0-9A-Za-z_+-])(?P<value>eyJ[a-zA-Z0-9=/-]{64,360})(?![=0-9A-Za-z_+-])
-  filter_type:
-    - ValueGrafanaCheck
-  min_line_len: 67
-  required_substrings:
-    - eyJ
-
-- name: Grafana Access Policy Token
-  severity: high
-  confidence: strong
-  type: pattern
-  values:
-    - (?<![0-9A-Za-z_+-])(?P<value>glc_eyJ[a-zA-Z0-9=/-]{80,360})(?![=0-9A-Za-z_+-])
-  filter_type:
-    - ValueGrafanaCheck
-  min_line_len: 87
-  required_substrings:
-    - glc_eyJ
-
-- name: Dropbox API secret (long term)
-  severity: high
-  confidence: weak
-  type: pattern
-  values:
-    - (?<![0-9A-Za-z_+-])(?=[A-Za-z0-9]{64})(?P<value>[A-Za-z0-9]{10,12}[B-Za-z0-9]A{10,12}[B-Za-z0-9][A-Za-z0-9]{40,44})(?![=0-9A-Za-z_+-])
-  filter_type: []
-  min_line_len: 43
-  required_substrings:
-    - AAAAAAAAAA
-
-- name: Dropbox App secret
-  severity: info
-  confidence: weak
-  type: pattern
-  values:
-    - (?<![.0-9A-Za-z_/+-])(?P<value>[a-z0-9]{15})(?![=0-9A-Za-z_/+-])
-  filter_type: WeirdBase36Token
-  min_line_len: 15
-  required_regex: "[a-zA-Z0-9_/+-]{15,80}"
-
-- name: Gitlab Incoming Email Token
-  severity: info
-  confidence: weak
-  type: pattern
-  values:
-    - (?<![.0-9A-Za-z_/+-])(?P<value>[a-z0-9]{24,25})(?![=0-9A-Za-z_/+-])
-  filter_type: WeirdBase36Token
-  min_line_len: 24
-  required_regex: "[a-zA-Z0-9_/+-]{15,80}"
-
-- name: Gitlab Feed Token
-  severity: info
-  confidence: weak
-  type: pattern
-  values:
-    - (?<![.0-9A-Za-z_/+-])(?P<value>[a-zA-Z0-9_-]{20})(?![=0-9A-Za-z_/+-])
-  filter_type: WeirdBase64Token
-  min_line_len: 20
-  required_regex: "[a-zA-Z0-9_/+-]{15,80}"
-
-- name: Jira 2FA
-  severity: info
-  confidence: weak
-  type: pattern
-  values:
-    - (?<![.0-9A-Za-z_/+-])(?P<value>[A-Z2-7]{16})(?![=0-9A-Za-z_/+-])
-  filter_type:
-    - ValueCoupleKeywordCheck
-    - ValuePatternCheck
-    - ValueEntropyBase32Check
-    - ValueBase32DataCheck
-    - ValueTokenBase32Check
-  min_line_len: 16
-  required_regex: "[a-zA-Z0-9_/+-]{15,80}"
-
-- name: OpenAI Token
-  severity: high
-  confidence: strong
-  type: pattern
-  values:
-    - (?<![.0-9A-Za-z_/+-])(?P<value>sk-\w{20}T3BlbkFJ\w{20})(?![=0-9A-Za-z_/+-])
-  min_line_len: 51
-  required_regex: T3BlbkFJ
-
-- name: Docker Swarm Token
-  severity: high
-  confidence: strong
-  type: pattern
-  values:
-    - (?<![.0-9A-Za-z_/+-])(?P<value>SWMTKN-1-[0-9a-z]{50}-[0-9a-z]{25})(?![=0-9A-Za-z_/+-])
-  min_line_len: 85
-  filter_type:
-    - ValueCoupleKeywordCheck
-  required_regex: SWMTKN-1-
+credsweeper_rule:
```

Comment:
The new level of config brokes many workflows :(
Do we need apply this?
BTW: CredSweeper has --rules param to load custom rules list

Reply:
Oh.. I didn't think about the workflows.. 
Hmm Okay I'll remove this level.

Reply:
Applied with 0d497a0, please check.